### ISSUE TITLE
Question (possible mistake) in Keras MelGAN tutorial

### ISSUE THREAD

**Issue by basbrs** [keras-team/keras-io#827]
This issue is about keras' (amazing, if i may add) [tutorial about the MelGAN audio synthesis network](https://keras.io/examples/audio/melgan_spectrogram_inversion/). 

In [line 387 and the following](https://github.com/keras-team/keras-io/blob/master/examples/audio/melgan_spectrogram_inversion.py) we define the generator's loss.

![grafik](https://user-images.githubusercontent.com/6853705/157840609-9d73dc29-ad5a-4738-9805-1c7481d2a609.png)

Why would we pass ground truth (`real_pred`) and generator prediction (`fake_pred`) while using only the `fake_pred` argument within the code? 
I'd assume that this is a mistake, because we wouldn't even be able to calculate any error this way, could we?

I'd appreciate your help or your clarification, have a nice day.
